server/database/seeds/items.js

```diff
@@ -0,0 +1,178 @@
+exports.seed = function(knex, Promise) {
+  // Deletes ALL existing entries
+  return knex('items')
+    .del()
+    .then(function() {
+      // Inserts seed entries
+      return knex('items').insert([
+        {
+          user_id: 1,
+          status_id: 1,
+          condition_id: 1,
+          category_id: 1,
+          subCategory_id: 1,
+          name: 'Insignia - 55" Class - LED - Smart - 4K UHD TV with HDR - Fire TV Edition',
+          quantity: 1,
+          dimensions: '55 inches',
+          viewCount: 170,
+          price: 339.0,
+          description: 'kick ass 55 inch tv bruh',
+          approved: true,
+        },
+        {
+          user_id: 1,
+          status_id: 2,
+          condition_id: 2,
+          category_id: 1,
+          subCategory_id: 1,
+          name: 'LG - 70" Class - LED - UK6190 Series - 2160p - Smart -4K UHD TV with HDR',
+          quantity: 2,
+          dimensions: '70 inches',
+          viewCount: 170,
+          price: 523.0,
+          description: 'ho cuz got one kick ass 70 inch tv cuz',
+          approved: false,
+        },
+        {
+          user_id: 2,
+          status_id: 1,
+          condition_id: 3,
+          category_id: 1,
+          subCategory_id: 2,
+          name: 'LG - Stylo 4 with 32GB Memory',
+          quantity: 3,
+          dimensions: '6.2"',
+          viewCount: 88,
+          price: 149.99,
+          description: 'Black phone with stylus',
+          approved: true,
+        },
+        {
+          user_id: 2,
+          status_id: 2,
+          condition_id: 4,
+          category_id: 1,
+          subCategory_id: 2,
+          name: 'Moto G6 - 32GB Black',
+          quantity: 900,
+          dimensions: '5.7"',
+          viewCount: 96,
+          price: 179.99,
+          description: 'lightly damaged but still works',
+          approved: false,
+        },
+        {
+          user_id: 3,
+          status_id: 1,
+          condition_id: 5,
+          category_id: 2,
+          subCategory_id: 1,
+          name: "Skechers Men's Go Walk Evolution Ultra-Impeccable Sneaker",
+          quantity: 4,
+          dimensions: 'Size 7',
+          viewCount: 17,
+          price: 17.17,
+          description: 'kick ass 55 inch tv bruh',
+          approved: true,
+        },
+        {
+          user_id: 3,
+          status_id: 2,
+          condition_id: 6,
+          category_id: 2,
+          subCategory_id: 1,
+          name: 'Adidas Womens Questar Slide',
+          quantity: 1,
+          dimensions: 'Size 5',
+          viewCount: 909,
```

Comment:
would it have any views if it "approved": false ?

Reply:
I was working under the assumption that all items being posted were approved until flagged for being inappropriate/vulgar and then set as approved: false from there.